api_spec/edge_api.json

```diff
@@ -91,6 +94,40 @@
                     }
                 }
             }
+        },
+        "/bulk-identities": {
+            "post": {
+                "tags": [],
+                "summary": "Bulk insert identities",
+                "description": "Bulk uploading identities and traits",
```

Comment:
I've just realised we should add here that it has a limit of 100 identities per request.